### PR TITLE
[MPA Classification] Enable Saliency Map Output

### PR DESCRIPTION
**Pull request by eugene123tw**
Related PRs: 
* https://github.com/openvinotoolkit/deep-object-reid/pull/224
* https://github.com/openvinotoolkit/model_preparation_algorithm/pull/30
* https://github.com/openvinotoolkit/model_preparation_algorithm/pull/34

### CODE REVIEW

**Comment by eugene123tw** [comment]
run ote-test

**Comment by eugene123tw** [comment]
run ote-test

File: external/deep-object-reid/torchreid_tasks/inference_task.py [hunk 1]
```diff
@@ -295,7 +293,8 @@ def export(self, export_type: ExportType, output_model: ModelEntity):
                     self._model.old_forward = self._model.forward
                     self._model.forward = lambda x: self._model.old_forward(x, return_all=True, apply_scale=True)
                     export_onnx(self._model.eval(), self._cfg, onnx_model_path,
-                                opset=self._cfg.model.export_onnx_opset, output_names=['logits', 'features', 'vector'])
+                                opset=self._cfg.model.export_onnx_opset, output_names=[
+                                  'logits', 'saliency_map', 'feature_vector'])
```
Comment (goodsong81) [inline external/deep-object-reid/torchreid_tasks/inference_task.py:297]: You reordered vector & map in MPA. Don't we need to align the order here?

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Please resolve conflict.

File: external/deep-object-reid/torchreid_tasks/inference_task.py (hunk 1 above)
Comment (eugene123tw) [reply external/deep-object-reid/torchreid_tasks/inference_task.py:297]: No, the order is correct for torchreid export. Here's the PR for torchreid: https://github.com/openvinotoolkit/deep-object-reid/pull/224/files#diff-5c0de0a508fdcc048ca43fa7a53b36e2a2e80afb149a4346371a05f85d78c198R392

**Review by goodsong81 — APPROVED** [review]
LGTM

**Review by JihwanEom — APPROVED** [review]
LGTM

**Pull request merged**